src/controllers/capprevision_test.go

```diff
@@ -0,0 +1,180 @@
+package controllers_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/dana-team/container-app-operator/api/v1alpha1"
+	"github.com/dana-team/platform-backend/src/controllers"
+	"github.com/dana-team/platform-backend/src/controllers/mocks"
+	"github.com/dana-team/platform-backend/src/types"
+	"github.com/dana-team/platform-backend/src/utils"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+const (
+	cappRevisionNamespace = testName + "-capp-revision-ns"
+	cappRevisionName      = testName + "-capp-revision"
+	labelKey              = "key"
+	labelValue            = "value"
+)
+
+var (
+	controller       controllers.CappRevisionController
+	cappRevision     v1alpha1.CappRevision
+	cappRevisionList types.CappRevisionList
+)
+
+func setupCappRevisions() {
+	createTestNamespace(cappRevisionNamespace)
```

Comment:
all tests should run isolated as they can.

isolation is very important concept in unit testing

https://medium.com/@tasdikrahman/f-i-r-s-t-principles-of-testing-1a497acda8d6